src/components/Pivot.tsx

```diff
@@ -239,7 +241,7 @@ export default function PivotTable({ id }: { id: number }) {
                   {columns.map((column) => renderColumnBlock(column))}
                 </div>
               </div>
-              <div className="h-72 overflow-auto">
+              <div className="h-72 overflow-auto" key={JSON.stringify(gridData)}>
```

Comment:
can we use a better key than stringifying gridata? this is not performant at all

Reply:
![image](https://github.com/pretzelai/pretzelai/assets/113238477/317597aa-fabd-4d45-bf53-6a2e0900a6a0)


Is it okay to use `uuidv4()` or should I use a different method?

Reply:
keys should be consistent between re-renders. Do a bit of research on this topic and update your PR :)